src/vma/event/event_handler_manager.cpp

```diff
@@ -823,8 +823,14 @@ void* event_handler_manager::thread_loop()
 	int timeout_msec;
 	int maxevents = INITIAL_EVENTS_NUM;
 
-	struct epoll_event* p_events = (struct epoll_event*)malloc(sizeof(struct epoll_event)*maxevents);
 	struct pollfd poll_fd;
+	struct epoll_event* p_events = (struct epoll_event*)malloc(sizeof(struct epoll_event)*maxevents);	
+	BULLSEYE_EXCLUDE_BLOCK_START
+	if(!p_events){
```

Comment:
line formatting: add spaces in the 'if () {' statement 
see the rest of the code in function